src/main/java/org/folio/des/scheduling/quartz/QuartzExportJobScheduler.java

```diff
@@ -57,6 +57,25 @@ public List<Job> scheduleExportJob(ExportConfig exportConfig) {
     }
   }
 
+  @Override
+  public void deleteJobGroup(String tenantId) throws SchedulerException {
+    try {
+      String ediJobName = getJobGroup(tenantId);
+      Set<JobKey> jobKeySet = scheduler.getJobKeys(GroupMatcher.groupEquals(ediJobName));
+      for (JobKey jobKey : jobKeySet) {
+        scheduler.deleteJob(jobKey);
+      }
+      log.info("deleteJobGroup:: Scheduled Job Keys with size={} deleted", jobKeySet.size());
+    } catch (SchedulerException e) {
+      log.error(e.getMessage(), e);
+      throw e;
+    }
+  }
+
+  private String getJobGroup(String tenantName) {
+    return tenantName + "_" + QuartzConstants.EDIFACT_ORDERS_EXPORT_GROUP_NAME;
```

Comment:
This class provides general functionality for scheduling based on exportConfig. It should not be tied to concrete export type. Should be changed to be provided by its users, either as a method parameter, or can be set in constructor.